recipe/conda_build_config.yaml

```diff
@@ -354,7 +354,7 @@ arb:
 arpack:
   - 3.6.3
 arrow_cpp:
-  - 0.15.1
+  - 0.16.0
```

Comment:
Note that we have a `run_exports` on `x.x.x`, this was wrong in the migration but didn't lead to harm as there was no new arrow-cpp release.